website/source/docs/job-specification/ephemeral_disk.html.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: "docs"
+page_title: "ephemeral_disk Stanza - Job Specification"
+sidebar_current: "docs-job-specification-ephemeral_disk"
+description: |-
+  The "ephemeral_disk" stanza instructs Nomad to utilize an ephemeral disk
+  instead of a hard disk requirement, and can also enable sticky volumes and
+  live data migrations.
+---
+
+# `ephemeral_disk` Stanza
+
+<table class="table table-bordered table-striped">
+  <tr>
+    <th width="120">Placement</th>
+    <td>
+      <code>job -> group -> **ephemeral_disk**</code>
+    </td>
+  </tr>
+</table>
+
+The `ephemeral_disk` stanza instructs Nomad to utilize an ephemeral disk instead
+of a hard disk requirement. Clients using this stanza should not specify disk
```

Comment:
Remove hard disk requirement part. Reuse what I have above.

I think you may not have seen this but tasks don't specify disk requirements any more. It is just at the task group level now.